climate.py

```diff
@@ -403,21 +403,26 @@ def get_scouting_observation_attachments(token,
 
 
 def log_http_error(response):
-    if response.status_code == 403:
+
+    """
+    Private function to log errors on server console
+    :param response: http response object.
+    """
+    if response.text.status_code == 403:
         Logger().error("Permission error, current scopes are - {}".format(
             os.environ['CLIMATE_API_SCOPES']))
-    elif response.status_code == 400:
-        Logger().error("Bad request - {}".format(response.json()))
-    elif response.status_code == 401:
-        Logger().error("Unauthorized - {}".format(response.json()))
-    elif response.status_code == 404:
-        Logger().error("Resource not found - {}".format(response.json()))
-    elif response.status_code == 416:
-        Logger().error("Range Not Satisfiable - {}".format(response.json()))
-    elif response.status_code == 500:
-        Logger().error("Internal server error - {}".format(response.json()))
-    elif response.status_code == 503:
-        Logger().error("Server busy - {}".format(response.json()))
+    elif response.text.status_code == 400:
```

Comment:
Is this right? I think it's just `response.status_code`

Reply:
No - this is wrong. I will fix this. I tested the code but I guess did not hit this path.